R/pkg/R/DataFrame.R

```diff
@@ -2950,6 +3038,10 @@ setMethod("drop",
           })
 
 # Expose base::drop
+#' @name drop
+#' @rdname drop
```

Comment:
this would add a fairly empty Rd page for drop.. I wonder if there is a way to avoid that? Perhaps add a link to base::drop?